modules/debts_api/lib/debts_api/v0/fsr_form_transform/additional_data_calculator.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'debts_api/v0/fsr_form_transform/expense_calculator'
+require 'debts_api/v0/fsr_form_transform/utils'
+
+module DebtsApi
+  module V0
+    module FsrFormTransform
+      class AdditionalDataCalculator
+        include ::FsrFormTransform::Utils
+
+        def initialize(form)
+          @form = form
+          @comments = @form.dig('additional_data', 'additional_comments')
+          @expense_calculator = DebtsApi::V0::FsrFormTransform::ExpenseCalculator.build(@form)
+        end
+
+        def get_data
+          {
+            'bankruptcy' => get_bankruptcy_data,
+            'additionalComments' => get_comments
+          }
+        end
+
+        def get_bankruptcy_data
+          # these should probably be digs
```

Comment:
Did you want to turn these into digs? Will there be errors if these values are nil? Maybe we want to raise a custom exception 🤔

Reply:
Yeah, right now if `@form['additional_data']['bankruptcy']` is nill we'll get an 'undefined method for nil' exception. We should probably come to some agreement with FE about our data contract generally.